Crane/app/src/main/java/androidx/compose/samples/crane/home/LandingScreen.kt

```diff
@@ -32,7 +32,7 @@ private const val SplashWaitTime: Long = 2000
 @Composable
 fun LandingScreen(modifier: Modifier = Modifier, onTimeout: () -> Unit) {
     Box(modifier = modifier.fillMaxSize(), alignment = Alignment.Center) {
-        LaunchedTask {
+        LaunchedEffect(Unit) {
```

Comment:
This will call the `onTimeout` from initial composition even if it changes.

Suggested changes: Either pass onTimeout to LaunchedEffect to restart the timer, or use `rememberUpdatedState` to safely capture `onTimeout`

Reply:
Sure! Done, thanks